### PR TITLE
React table

### PR DESCRIPTION
**Pull request by niksfred**
React table implemented for Clients page with Sorting and Search functionalities.

### CODE REVIEW

File: src/components/ClientsTable/index.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+  const { globalFilter } = state;
+
+  if (isLoading) return null;
+  return (
+    <>
+      <SearchField
+        filter={globalFilter}
+        setFilter={setGlobalFilter}
+        classes={"absolute top-24 bg-white p-1 border-2 rounded-lg"}
```
Comment (Galabov93) [inline src/components/ClientsTable/index.js:80]:
```suggestion
        classes="absolute top-24 bg-white p-1 border-2 rounded-lg"
```

Comment (Galabov93) [inline src/components/ClientsTable/index.js:80]:
```suggestion
        className={"absolute top-24 bg-white p-1 border-2 rounded-lg"}
```

**Review by Galabov93 — APPROVED** [review]
Well done

File: src/components/ClientsTable/index.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  const {
+    data: tableData,
+    status,
+    isLoading,
+  } = useQuery("clients", fetchClients);
+
+  const data = React.useMemo(() => {
+    return (
+      tableData?.data?.map((el) => ({
+        col1: el.name,
```
[the remark this answers is not in the document]
Comment (niksfred) [reply src/components/ClientsTable/index.js:17]: New push incoming.

File: src/components/ClientsTable/index.js (hunk 1 above)
Comment (niksfred) [reply src/components/ClientsTable/index.js:80]: Noted.

**Pull request merged**